pom.xml

```diff
@@ -2,7 +2,8 @@
 <!--
  DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 
- Copyright © 2011 ForgeRock AS. All rights reserved.
```

Comment:
Unintended copy right line change I think?

Reply:
it's intentional. this POM came from a FR source JAR.